src/main/java/twilight/AddCommand.java

```diff
@@ -0,0 +1,36 @@
+package twilight;
+
+import java.io.IOException;
+
+/**
+ * Represents a Command to add a particular instance of an event.
+ */
+public class AddCommand extends Command {
+    protected int type;
+    protected String details;
+
+    public AddCommand(int type, String details) {
+        this.type = type;
+        this.details = details;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Storage storage) throws InvalidInputException {
+        String addition = "";
```

Comment:
Is it possible to give the string a more descriptive name?

Reply:
For string that are part of a method I think there is more leeway so I will stick to this.